[2019-03-19 12:28:14] [error] [/home/hometqa/minganci_system/aikeyword/hmai/hmai_gate_keyword.py - 384 ] {'result': [{'code': '-1', 'mr': [], 'model': 'keyword', 'error': 'imgurl', 'error_mes': 'fail for geting image from urlï¼š at 2019-03-19-12:28:14 '}], 'pro_ver': '2.1'}
[2019-03-19 12:32:19] [error] [/home/hometqa/minganci_system/aikeyword/hmai/hmai_gate_keyword.py - 374 ] {'result': [{'code': '-1', 'mr': [], 'model': 'keyword', 'error': 'imgdata', 'error_mes': 'fail fot geting image from data at 2019-03-19-12:32:19 '}], 'pro_ver': '2.1'}
[2019-03-19 12:36:09] [error] [/home/hometqa/minganci_system/aikeyword/hmai/hmai_gate_keyword.py - 375 ] {'result': [{'code': '-1', 'mr': [], 'model': 'keyword', 'error': 'imgdata', 'error_mes': 'Traceback (most recent call last):\n  File "/home/hometqa/minganci_system/aikeyword/hmai/hmai_gate_keyword.py", line 345, in get_tasks\n    req_imgurl = urllib.request.unquote(request.form.get(\'imgurl\'))\n  File "/home/hometqa/anaconda3/envs/py36/lib/python3.6/urllib/parse.py", line 608, in unquote\n    if \'%\' not in string:\nTypeError: argument of type \'NoneType\' is not iterable\nfail fot geting image from data at 2019-03-19-12:36:09 '}], 'pro_ver': '2.1'}
